src/classes/renderer/pdf.vala

```diff
@@ -83,7 +83,9 @@ namespace pdfpc {
             if (this.cache != null) {
```

Comment:
Lets get rid of the other renderers as well. We are a pdf renderer, nothing more! :)